gkgen/bcp47gen_test.go

```diff
@@ -5,72 +5,70 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 )
 
 type TestData struct {
 	BCP47String *string
 }
 
-// BCP47ValidatorTestSuite
-type BCP47ValidatorTestSuite struct {
-	suite.Suite
-}
-
-// TestBCP47ValidatorTestSuite
-func TestBCP47ValidatorTestSuite(t *testing.T) {
-	suite.Run(t, new(BCP47ValidatorTestSuite))
-}
-
 // TestIsBCP47_ParamsLength
-func (s *BCP47ValidatorTestSuite) TestIsBCP47_ParamsLength() {
+func TestIsBCP47_ParamsLength(t *testing.T) {
```

Comment:
Seeing as (as far as I can tell) these tests no longer share any state, we can probably mark them all with `t.Parallel()` as well. I suspect it doesn't make a noticeable difference in speed, but might be worth doing anyway.